### PR TITLE
Introduce branding

### PR DESCRIPTION
**Pull request by notlee**
# Updates

- Introduce the concept of branding.
- Separate "row stripes" into a new demo.
- Remove a little duplicate CSS.

screen shot 2018-03-26 at 14 37 33

### CODE REVIEW

**Comment by notlee** [comment]
Worst commit messages in history here. 😅 
Thank you squash and merge. 🙏

**Comment by JakeChampion** [comment]
What is the status of this PR, are we merging it?

**Comment by notlee** [comment]
@JakeChampion No plan to merge at present. Will be running some workshops to gather feedback using this branch as a demo and source of examples --
 betas are tagged against this branch.

File: src/scss/_responsive-flat.scss [hunk 1]
```diff
@@ -5,64 +5,65 @@
...
+		tbody tr th {
+			border-right: 1px solid oBrandGet('table-border-color');
+			float: left;
+			padding: 8px;
+			width: 50%;
+
+			@include oGridRespondTo(S) {
+				display: none;
+			}
+		} // td th
```
Comment (gvonkoss) [inline src/scss/_responsive-flat.scss]: I thought these were accidental comments 🤔 I get that they're meant to indicate the end of a block, so I suppose this should be `tbody tr th`? Generally, though, it strikes me as a little odd if its not replicated across other files...Is there any reason for that?

**Comment by gvonkoss** [comment]
Looking 👌

File: src/scss/_responsive-flat.scss (hunk 1 above)
Comment (notlee) [reply src/scss/_responsive-flat.scss]: This was there before, it shows in the diff as I increased the indent. I agree it's useful but relies too much on developer discipline to use everywhere correctly. I decided not to remove but not to continue these kinds of comments.

**Pull request merged**